From: Eric Van Hensbergen <ericvh@gmail.com>
Subject: [PATCH 059/111]     fs/9p: remove sparse warning in vfs_inode
Git-commit: ea1375333ef58298ba4d3c638f3cba982c76504d
Patch-Mainline: 2.6.36
References: FATE#311639

    make v9fs_dentry_from_dir_inode static

	upstream commit ea1375333ef58298ba4d3c638f3cba982c76504d

    Signed-off-by: Eric Van Hensbergen <ericvh@gmail.com>
Signed-off-by: Harsh Prateek Bora <harsh@linux.vnet.ibm.com>
Acked-by: Bruce Rogers <brogers@suse.com>
---
 fs/9p/vfs_inode.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fs/9p/vfs_inode.c b/fs/9p/vfs_inode.c
index 902d02b..d97fe55 100644
--- a/fs/9p/vfs_inode.c
+++ b/fs/9p/vfs_inode.c
@@ -257,7 +257,7 @@ static gid_t v9fs_get_fsgid_for_create(struct inode *dir_inode)
  *
  */
 
-struct dentry *v9fs_dentry_from_dir_inode(struct inode *inode)
+static struct dentry *v9fs_dentry_from_dir_inode(struct inode *inode)
 {
 	struct dentry *dentry;
 
-- 
1.7.1.1

